README.adoc

```diff
@@ -145,306 +136,186 @@ The [hotspot=from file=0]`FROM` instruction initializes a new build stage and in
-`testServer`, then it can be started as shown in the following example:
+Results :
 
-[role='no_copy']
-```
-docker run -d --name rest-app -p 9080:9080 -p 9443:9443 -v <absolute path to guide>/start/target/liberty/wlp/usr/servers:/servers ol-runtime testServer
-```
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
+----
 
-Learn more about running containers on the https://docs.docker.com/engine/reference/run[official Docker website^]
+When you are finished, press `CTRL+C` in the session that the dev mode was started from. This will both stop and remove the container for you. 
+You can check that the container was stopped by running the `docker ps` command.
 
 // =================================================================================================
-// Testing the container
+// Starting development mode with run options 
 // =================================================================================================
 
-== Testing the container
+== Starting development mode with run options
 
-Before you access your application from the browser, run the `docker ps` command from the active
-command-line session to make sure that your container is running and didn't crash:
+Another useful feature of development mode with a container is the ability to pass additional options
+to the `docker run` command. You can do this by adding the `<dockerRunOpts>` tag to the `pom.xml` file under 
+the `<configuration>` tag of the Liberty Maven Plugin. Here is an example of an environment variable 
+being passed in:
 
 [role="no_copy"]
 ----
-$ docker ps
-CONTAINER ID        IMAGE               CREATED             STATUS              NAMES
-2720cea71700        ol-runtime          2 seconds ago       Up 1 second         rest-app
-----
-
-To view a full list of all available containers, run the `docker ps -a` command in the command-line session.
-
-If your container is running without problems, point your browser to http://localhost:9080/LibertyProject/System/properties[http://localhost:9080/LibertyProject/System/properties^],
-where you can see a JSON file that contains the system properties of the JVM in your container.
-[role="code_command hotspot", subs="quotes"]
-----
-#Update the `PropertiesResource` class.#
-`src/main/java/io/openliberty/guides/rest/PropertiesResource.java`
+<groupId>io.openliberty.tools</groupId>
+<artifactId>liberty-maven-plugin</artifactId>
+<version>3.3.2</version>
+<configuration>
+    <dockerRunOpts>-e ENV_VAR=exampleValue</dockerRunOpts>
+</configuration>
 ----
 
-[role="edit_command_text"]
-Change the endpoint of your application from `properties` to `properties-new` by changing the [hotspot=Path]`@Path`
-annotation to `"properties-new"`.
+If the Dockerfile isn't located in the directory that the `devc` goal is being
+run from, you can add the `<dockerfile>` tag to specify the location. Note that
+using this parameter will set the context for building the Docker image to the
```

Comment:
Change "Note that using this parameter will set the context" to "Using this parameter sets the context", to avoid future tense.